nrt/data/SMIB/RegulationProportional/SMIB_1_StepPm/SMIB.dyd

```diff
@@ -0,0 +1,50 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!--
+    Copyright (c) 2015-2019, RTE (http://www.rte-france.com)
+    See AUTHORS.txt
+    All rights reserved.
+    This Source Code Form is subject to the terms of the Mozilla Public
+    License, v. 2.0. If a copy of the MPL was not distributed with this
+    file, you can obtain one at http://mozilla.org/MPL/2.0/.
+    SPDX-License-Identifier: MPL-2.0
+
+    This file is part of Dynawo, an hybrid C++/Modelica open source time domain
+    simulation tool for power systems.
+-->
+<dyn:dynamicModelsArchitecture xmlns:dyn="http://www.rte-france.com/dynawo">
+  <!-- Network model -->
+  <dyn:modelicaModel id="Network">
+    <dyn:unitDynamicModel id="busInf" name="Dynawo.Electrical.Buses.InfiniteBus" parFile="SMIB.par" parId="1"/>
+    <dyn:unitDynamicModel id="line1" name="Dynawo.Electrical.Lines.Line" parFile="SMIB.par" parId="2"/>
+    <dyn:unitDynamicModel id="line2" name="Dynawo.Electrical.Lines.Line" parFile="SMIB.par" parId="3"/>
+    <dyn:unitDynamicModel id="tfo" name="Dynawo.Electrical.Transformers.TransformerFixedRatio" parFile="SMIB.par" parId="4"/>
+    <dyn:connect id1="busInf" var1="terminal" id2="line1" var2="terminal1"/>
+    <dyn:connect id1="busInf" var1="terminal" id2="line2" var2="terminal1"/>
+    <dyn:connect id1="line1" var1="terminal2" id2="tfo" var2="terminal1"/>
+    <dyn:connect id1="line2" var1="terminal2" id2="tfo" var2="terminal1"/>
+  </dyn:modelicaModel>
+  <!-- Generator with four windings with fixed excitation voltage and proportional speed regulation -->
```

Comment:
It seems that both the speed regulation and the excitation voltage are proportional regulations

Reply:
fixed